poetry.lock

```diff
@@ -1407,8 +1407,13 @@ name = "parso"
 version = "0.8.1"
 description = "A Python Parser"
 category = "main"
+<<<<<<< HEAD
```

Comment:
you might want to double check what happened here, still figuring things out a bit with poetry @SimonBiggs